src/system/Nav/NavItem.tsx

```diff
@@ -0,0 +1,80 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import React, { Ref, forwardRef } from 'react';
+import * as NavigationMenu from '@radix-ui/react-navigation-menu';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { VIP_NAV } from '.';
+import { Theme, ThemeUIStyleObject } from 'theme-ui';
+import { NavProps } from './Nav';
+
+interface NavItemTheme extends Theme {
+	outline?: Record< string, string >;
+}
+export interface NavItemProps extends NavigationMenu.NavigationMenuLinkProps {
+	className?: string;
+	disabled?: boolean | undefined;
+	sx?: ThemeUIStyleObject | undefined;
+	variant?: NavProps[ 'variant' ];
+}
+
+const styles = variant => ( {
+	backgroundColor: 'transparent',
+	mr: 0,
+	fontSize: 2,
+	borderRadius: 1,
+	p: 3,
+	px: 4,
+	display: 'inline-block',
+	color: 'link',
+	textDecoration: 'none',
+	'&[data-active]': {
+		color: `button.${ variant }.label.default`,
+		backgroundColor: `button.${ variant }.background.default`,
+		fontWeight: 'regular',
+		boxShadow: 'inset 0 -1px 0 0',
+	},
+	'&[aria-disabled="true"]': {
+		opacity: 0.7,
+		color: 'texts.secondary',
+		cursor: 'not-allowed',
+	},
+	':hover': {
+		fontWeight: 'regular',
+		color: `button.${ variant }.label.hover`,
+		backgroundColor: `button.${ variant }.background.hover`,
+		textDecoration: 'none',
+	},
+	'&:focus': ( theme: NavItemTheme ) => theme.outline,
+	'&:focus-visible': ( theme: NavItemTheme ) => theme.outline,
+} );
+
+const NavItem = forwardRef< HTMLAnchorElement, NavItemProps >(
+	(
+		{ className, children, href, active, disabled, variant = 'primary' }: NavItemProps,
+		ref: Ref< HTMLAnchorElement >
+	) => {
+		return (
+			<NavigationMenu.Item className={ classNames( `${ VIP_NAV }-item`, className ) }>
+				<NavigationMenu.Link
```

Comment:
we need a way to pass a framework Link component (e.g., `next/link`)

Reply:
We are working on it: https://www.radix-ui.com/primitives/docs/components/navigation-menu#with-client-side-routing. I think this should work.

Reply:
That means we need to export `NavigationMenu.Link`, right?

Reply:
@chriszarate It already works like this:

```
<>
					<Nav.Root label="SSO" variant="tabs">
						{ TABS.map( tab => (
							<Nav.Item
								key={ tab.path }
								active={ tab.path === activeTab.path }
								href={ tab.path }
								asChild
							>
								<Link
									href={ `/orgs/${ id }/sso/configurations/${ idP }/edit/${ tab.path }` }
								>
									{ tab.name }
								</Link>
							</Nav.Item>
						) ) }
					</Nav.Root>
					<Box>
						<section sx={ { maxWidth: '750px' } }>
							{ ! loadingData && ! hasSSOPermission ? (
								<ConfigureSSOPermissionDenied />
							) : (
								activeTab.content
							) }
						</section>
					</Box>
				</>
```